holoviews/element/stats.py

```diff
@@ -34,7 +34,8 @@ def __init__(self, data, kdims=None, vdims=None, **params):
         elif len(vdims) > 1:
             raise ValueError(f"{type(self).__name__} expects at most one vdim.")
         else:
-            self.vdims = process_dimensions(None, vdims)['vdims']
+            with param.edit_constant(self):
+                self.vdims = process_dimensions(None, vdims)['vdims']
```

Comment:
```suggestion
            self.vdims = process_dimensions(None, vdims)['vdims']
```

Related to: https://github.com/holoviz/holoviews/pull/6545